src/transformers/modeling_distilbert.py

```diff
@@ -503,10 +503,9 @@ def forward(
         attention_mask=None,
         head_mask=None,
         inputs_embeds=None,
-        masked_lm_labels=None,
```

Comment:
I think `masked_lm_labels` was deleted accidently here

Reply:
Renamed to `labels`